### DISCUSSION TITLE
Threshold on bar charts?

### DISCUSSION THREAD

**Discussion by RyuunosukeDS3** [grafana/grafana#39176] [Dashboards, Panels & Visualizations]
I can't ad threshold lines on barchart, is it not supported yet? Like this, for example:

![image](https://user-images.githubusercontent.com/31015675/133253232-1b7127de-aab1-4de6-a267-d149f42a07fd.png)

It doesn't seem to have the option:

![image](https://user-images.githubusercontent.com/31015675/133253315-3e78ac04-f7c1-4ff7-95be-a7bd619e6cf1.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
